test/tools/mongodb-mock/src/server.js

```diff
@@ -108,7 +108,7 @@ class MockServer extends EventEmitter {
           // this is when the driver closes connections
           // change false to true to start printing the logs,
           // keep CI logs clear for test reporting
-          false && console.warn('connection error: ', e);
+          console.warn('connection error: ', e);
```

Comment:
Why change to always print the logs?

Reply:
This causes a lint failure, because the condition is redundant - we never log anything.  Alternatively, I can just remove this log entirely.  Do you have a preference?

Reply:
No preference, was just curious.